packages/server-admin-ui/src/views/ServerConfig/ServerLogs.js

```diff
@@ -0,0 +1,143 @@
+import React, { Component } from 'react'
+import ReactHtmlParser from 'react-html-parser'
+import { connect } from 'react-redux'
+import {
+  Badge,
+  Button,
+  Card,
+  CardHeader,
+  CardBody,
+  CardFooter,
+  InputGroup,
+  InputGroupAddon,
+  Input,
+  Form,
+  Col,
+  Label,
+  FormGroup,
+  FormText,
+  Table
+} from 'reactstrap'
+
+class ServerLogs extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      hasData: true
+    }
+
+    this.handleDebug = this.handleDebug.bind(this)
+  }
+
+  componentDidMount () {
```

Comment:
If you navigate directly to /admin/#/serverConfiguration/log or hit reload the component is mounted before the ws is open and subscribe never happens.

Also after a server restart the log subscribe never happens if you have the page open even if the ws reconnects.